modules/rubiconBidAdapter.js

```diff
@@ -93,32 +93,21 @@ export const spec = {
       return false;
     }
 
-    // Log warning if context is 'outstream', is not currently supported
-    if (utils.deepAccess(bid, `mediaTypes.${VIDEO}.context`) === 'outstream') {
-      utils.logWarn('Warning: outstream video for Rubicon Client Adapter is not supported yet');
-    }
-
-    // Log warning if mediaTypes contains both 'banner' and 'video'
-    if (spec.hasVideoMediaType(bid) && typeof utils.deepAccess(bid, `mediaTypes.${BANNER}`) !== 'undefined') {
-      utils.logWarn('Warning: instream video and banner requested for same ad unit, continuing with video instream request');
-    }
-
-    // Bid is invalid if legacy video is set but params video is missing size_id
-    if (bid.mediaType === 'video' && typeof utils.deepAccess(bid, 'params.video.size_id') === 'undefined') {
-      return false;
-    }
-
-    // Bid is invalid if mediaTypes video is invalid and a mediaTypes banner property is not defined
-    if (bid.mediaTypes && !spec.hasVideoMediaType(bid) && typeof bid.mediaTypes.banner === 'undefined') {
-      return false;
-    }
-
-    let parsedSizes = parseSizes(bid);
-    if (parsedSizes.length < 1) {
+    if (hasVideoMediaType(bid)) {
+      // Log warning if mediaTypes contains both 'banner' and 'video'
+      if (typeof utils.deepAccess(bid, `mediaTypes.${BANNER}`) !== 'undefined') {
+        utils.logWarn('Warning: instream video and banner requested for same ad unit, continuing with video instream request');
+      }
+      // Bid is invalid if legacy video is set but params video is missing size_id
+      if (typeof utils.deepAccess(bid, 'params.video.size_id') === 'undefined') {
```

Comment:
If the bid is `outstream` we are later defaulting it to be `size_id` `203` later anyways.

Let's make this check strictly for instream, and if it is outstream without a size_id, we can just let it defualt itself to `203` like it does on line 149